### PR TITLE
Update good first template

### PR DESCRIPTION
**Pull request by RainbowMango**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
The `/assign` command in a separated line will assign the issue to the author. This PR fixed this issue.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/good-first.md [hunk 1]
```diff
@@ -14,17 +14,15 @@ labels: good first issue
 
 **Who can join or take the task**:
 
-Good first issue is intended for `first-time contributors` to get started on his/her contributor journey.
+The good first issue is intended for `first-time contributors` to get started on his/her contributor journey.
 
-After a contributor has successfully completed 1-2 good first issue's, 
-they should be ready to move on to `help wanted` items, saving remaining `good first issue`'s for other new contributors.
+After a contributor has successfully completed 1-2 good first issue's,
+they should be ready to move on to `help wanted` items, saving the remaining `good first issues for other new contributors.
```
Comment (XiShanYongYe-Chang) [inline .github/ISSUE_TEMPLATE/good-first.md:20]: There's a missing backquote:
 
... saving the remaining `good first issues` for other new contributors.

Comment (RainbowMango) [reply .github/ISSUE_TEMPLATE/good-first.md]: right, done.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Pull request merged**